frontend/modelarchive/src/test/java/com/amazonaws/ml/mms/archive/Exporter.java

```diff
@@ -121,17 +121,17 @@ public static void main(String[] args) {
 
             String runtime = config.getRuntime();
             if (runtime != null) {
-                manifest.getEngine().setRuntime(Manifest.RuntimeType.fromValue(runtime));
+                manifest.setRuntime(Manifest.RuntimeType.fromValue(runtime));
             }
 
             File symbolFile = findUniqueFile(files, "-symbol.json");
             if (symbolFile != null) {
-                model.setSymbolFile(symbolFile.getName());
+                model.addExtension("symbolFile", symbolFile.getName());
             }
 
             File paramsFile = findUniqueFile(files, ".params");
             if (paramsFile != null) {
-                model.setParametersFile(paramsFile.getName());
+                model.addExtension("parametersFile", paramsFile.getName());
```

Comment:
IS there a way for customers to feed in the symbol/params file? This seems like we are assuming we will have only one params/symbol file or else we throw an exception.

Reply:
1. This is an unit test code.
2. This logic is the same as current python export logic. I think we should allow user to specify these values, but I preferred do it with a UI.